pymc3/step_methods/hmc/nuts.py

```diff
@@ -110,7 +130,17 @@ def astep(self, q0):
 
         self.m += 1
 
-        return q
```

Comment:
We should also add the divergence -- the difference between the logp at the old and proposed position.